Examples/Blink/Core/Src/main.cpp

```diff
@@ -21,9 +21,12 @@
 #include "main.h"
 #include "usart.h"
 #include "gpio.h"
+#include "spi.h"
 
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
+#include "Peripherals/inc/SPIInterface.h"
+#include <cstdint>

```

Comment:
Typically the user includes come after stdlib includes. I will get you to put cstdint above SPIInterface.h, though I think we may eventually want to scrap their boilerplate main file and create our own, tidier one.

Reply:
Wasn't even necessary,  just deleted.